src/autoloads/resources.gd

```diff
@@ -0,0 +1,71 @@
+extends Node
+
+## HOW TO USE THE RESOURCE MANAGER
+## Example: get a list of resources in a set of directories
+## Dictionary of directories, with the namespace, followed by the directory path:
+#	var dirs: Dictionary = {
+#	"Resource 1": "res://examples/resource1",
```

Comment:
I feel like using "Folder 1" or "Directory 1" would be more readable than "Resource 1," because in Godot a resource is an instance of an actual resource class, like a Texture. I think changing variable names away from `resource` and `resources` in `list()` and `_filesToDictionary()` to describe that they are names would also help.

Reply:
yeah good call implemented